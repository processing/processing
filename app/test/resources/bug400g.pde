////
void setup(){
  size(100,100);

    if(true){
    }
    else{ // Syntax error on token "else", } expected
    }
}