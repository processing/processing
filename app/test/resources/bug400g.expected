import processing.core.*; 
import processing.data.*; 
import processing.event.*; 
import processing.opengl.*; 

import java.util.HashMap; 
import java.util.ArrayList; 
import java.io.File; 
import java.io.BufferedReader; 
import java.io.PrintWriter; 
import java.io.InputStream; 
import java.io.OutputStream; 
import java.io.IOException; 

public class bug400g extends PApplet {

////
public void setup(){
  size(100,100);

    if(true){
    }
    else{ // Syntax error on token "else", } expected
    }
}
    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "bug400g" };
        if (passedArgs != null) {
          PApplet.main(concat(appletArgs, passedArgs));
        } else {
          PApplet.main(appletArgs);
        }
    }
}
